src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -128,13 +128,6 @@
           "size.")                                                          \
           range(0,100)                                                      \
                                                                             \
-  product(uintx, ShenandoahOldMinFreeThreshold, 5, EXPERIMENTAL,            \
-          "Percentage of free old generation heap memory below which most " \
-          "heuristics trigger collection independent of other triggers. "   \
-          "Provides a safety margin for many heuristics. In percents of "   \
-          "(soft) max heap size.")                                          \
-          range(0,100)                                                      \
-                                                                            \
```

Comment:
Would it be correct to assume that this behaviour is no longer present or, if present, no longer directly controllable in this manner?

It'd be great to have a PR comment against each of these knob changes motivating the changes to their default values, or of their no longer being available.